client/src/components/classifiedCard/index.jsx

```diff
@@ -0,0 +1,86 @@
+import { useState, useEffect } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import axios from 'axios';
+import * as M from '../../mui-modules';
+
+import './style.css';
+
+const CalssifiedCard = () => {
+  const [data, dataSet] = useState([]);
+  const [message, setMessage] = useState();
+  const { search } = useParams();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const getData = async () => {
+      const resultes = await axios.get(`/api/v1/products/search?q=${search}`);
+      const res = resultes.data;
+      if (typeof (res) === 'undefined') {
```

Comment:
I think we can simply  describe it like this 
```suggestion
      if (!res) {
```